tests/lib/rules/prefer-find-by.test.ts

```diff
@@ -702,6 +702,29 @@ ruleTester.run(RULE_NAME, rule, {
 							queryMethod
 						)}('foo', { name: 'baz' })
           })
+        `,
+		})),
+		// Issue #579, https://github.com/testing-library/eslint-plugin-testing-library/issues/579
+		// findBy can have two sets of options: await screen.findByText('text', queryOptions, waitForOptions)
+		...createScenario((waitMethod: string, queryMethod: string) => ({
```

Comment:
Thanks to this, I realized this rule (and other parts of the plugin) are still checking for wait methods not supported anymore! I've created https://github.com/testing-library/eslint-plugin-testing-library/issues/681 to remove this in the next major.